api-reference/echo-node-api/database-api.md

```diff
@@ -177,51 +190,194 @@ If *clear_filter* is set to true, the API server will notify all newly created o
+                        21,
+                        {
+                            "fee": {
+                                "amount": 0,
+                                "asset_id": "1.3.0"
+                            },
+                            "deposit_to_account": "1.2.26",
+                            "balance_to_claim": "1.8.0",
+                            "balance_owner_key": "ECHO6XS3BMVnEHAzo1PhHWt9vndrZn2P27tCbU9WdqCM8sJu",
+                            "total_claimed": {
+                                "amount": "1000000000000000",
+                                "asset_id": "1.3.0"
+                            },
+                            "extensions": []
+                        }
+                    ]
+                ],
+                "extensions": [],
+                "signatures": [
+                    "1c8747b07f8131b4caaa52932c14a2472b52fdff339456ceb52befe5f2f14142e0020a4ba02258a68c43668bd36fc4c56ba19234a9a525e9c493fbc251103e0a"
+                ],
+                "signed_with_echorand_key": false
+            }
+        ]
+    ]
+}
+```
+
+#### Returns
 
-| Option                                     | Description                                         |
-|--------------------------------------------|:----------------------------------------------------|
-| `function<void(const variant&)> callback`  | notifications for incoming unconfirmed transactions |
+```json
+{
+    "id": 4,
+    "jsonrpc": "2.0",
+    "result": null
+}
+```
 
-#### set_block_applied_callback(callback)
+### set_block_applied_callback(callback)
 
 Subscribe to block applications.
 
-##### Parameters
+#### Parameters
+
+| Option                                      | Description                                                                  |
+|---------------------------------------------|:-----------------------------------------------------------------------------|
+| `function<void(variant block_id)> callback` | gives a notification whenever the block blockid is applied to the blockchain |
+
+#### Example
+
+```json
+{
+    "id": 4,
+    "method": "call",
+    "params": [
+        DATABASE_API_ID,
+        "set_block_applied_callback",
+        [
+            CALLBACK_ID
+        ]
+    ]
+}
+```
+
+#### Notice example
+
+```json
+{
+    "method": "notice",
+    "params": [
+        CALLBACK_ID,
+        [
+            "0013191865d4306288d52d2f648476508a159a0d"
+        ]
+    ]
+}
+```
+
+#### Returns
 
-| Option                                              | Description                                                                  |
-|-----------------------------------------------------|:-----------------------------------------------------------------------------|
-| `function<void(const variant& block_id)> callback`  | gives a notification whenever the block block_id is applied to the blockchain |
+```json
+{
+    "id": 4,
+    "jsonrpc": "2.0",
+    "result": null
+}
+```
 
 #### cancel_all_subscriptions()
```

Comment:
change to `### cancel_all_subscriptions()`

Reply:
Fixed